motion/p5js/raindrops/index.md

```diff
@@ -0,0 +1,12 @@
+---  
+layout: p5js
+author: Julian Broocks
+author-url: https://github.com/hulianbrox
+title: Raindrops
+tags: 
+  - P5.js
+  - motion
+  - Law of Similarity
+permalink: /motion/raindrops/
+summary: Elements of same speed towards same direction are seen as grouped together.
```

Comment:
Please make this a real sentence:  

 Elements of same speed towards **the** same direction are seen as a group ~~grouped together~~.